i3df/build.rs

```diff
@@ -329,7 +329,7 @@ fn main() -> Result<(), Box<dyn Error>> {
         #[derive(Clone, Copy, Debug, Default, Deserialize, Serialize)]
         pub struct Texture {
             // TODO generate from YAML
-            pub file_id: u64,
+            pub file_id: u32,
```

Comment:
Just a general comment:
The file_ids have previously been actual file_ids, which might require 53-bit precision. I think they must be exposed as f64 rather than u32 in that case. However, since we are moving towards files like `mesh_x.cmt` where `x` is the `file_id`, I think we could do the same for textures. That is, `texture_x.png` where `x` is in the range for `u32`.

Reply:
I think actually we _are_ doing the same for textures. But might be a good idea to use f64 here for consistency, @dragly ?

Reply:
Hm, yes, we should probably either user f64 here or change the others to u32. I agree that consistency is most important.